src/index.ts

```diff
@@ -320,35 +326,30 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
       const hasExistingFlags = !!Object.keys(evaluations).length
 
       // When authentication is done, fetch all flags
-      fetchFlags()
-        .then(() => {
-          logDebug('Fetch all flags ok', storage)
-        })
-        .then(() => {
-          if (closed) return
-          startStream() // start stream only after we get all evaluations
-        })
-        .then(() => {
-          if (closed) return
-
-          logDebug('Event stream ready', { storage })
```

Comment:
This was a useless log. There was no validation to check if streaming was ready. We already log the status of stream init in the streaming code any way.